site/en/Test.md

```diff
@@ -0,0 +1,41 @@
+
+During the test phase, the various options for installing the OCR-D software with and without a docker container were tried out and the software was successfully installed on a wide range of differently powerful, partly virtual servers. For non-Intel computers (ARM, PowerPC64) this was more complicated and time consuming, as individual Python packages were not executable on these computers and had to be adapted manually. The complete installation of all available OCR-D processors (``ocrd_all``), which was developed during the test phase, was confirmed as the simplest and least complex variant and was therefore the most recommendable. No pilot library integrated the OCR-D software In a workflow software such as Kitodo, as this would have meant too much work for a simple test run. 
+
+The usage of the numerous OCR-D processors was described as a challenge. Calling the processors was not so much of a problem than understanding their respective areas of application and, in particular, the selecting and combining the processors to form meaningful workflows. For the first test, besides the technical documentation of the software, there was no overall documentation on its use available yet, which is also aimed at users inexperienced in OCR. The requirements and wishes of the testers for such a documentation were taken into account when preparing the manuals, which are now available [in the user area of the OCR-D website](https://ocr-d.de/de/use.html). 
+
+The OCR-D software runs very stable, no library reported any aborts. All of the required output formats are already offered, whereas the few changes, which are still on the interfaces, are already planned for the further development of the prototype.
+
+The recognition quality was only checked on individual pages by the respective pilot libraries, as there is no ground truth available for the test books. Overall, the results of this first test run are promising. The Mannheim University Library has tested OCR-D with a focus on tesseract processors on five prints from the 16th to 19th century. On antiqua prints from the 17th and 18th centuries and a blackletter text from the 19th century, as expected, the best results of - in the case of the antiqua - significantly less than 0.1 CER were achieved for the raw data, whereas the blackletter prints from the 17th century was slightly above 0.1 CER. The greatest challenge was the 16th century bleckletter, where only a CER of just under 0.16 was reached. The BBAW provides a comprehensive insight into their testing, by making their [report and data publicly available](https://github.com/tboenig/ocrd_bbaw_pilotbibliothek).
```

Comment:
ground truth -> Ground Truth

tesseract -> Tesseract

Probably best to keep "fraktur" as-is rather than translate it to blackletter. But definitely not `bleckletter` :D